src/Sepes.Infrastructure/Service/Azure/StorageAccount/AzureStorageAccountTokenService.cs

```diff
@@ -48,6 +48,11 @@ public async Task<UriBuilder> CreateFileUploadUriBuilder(string containerName, C
             return await CreateUriBuilderWithSasToken(containerName, permission: BlobContainerSasPermissions.Write, expiresOnMinutes: 30, cancellationToken: cancellationToken);
         }
 
+        public async Task<UriBuilder> CreateFileDeleteUriBuilder(string containerName, CancellationToken cancellationToken = default)
+        {
+            return await CreateUriBuilderWithSasToken(containerName, permission: BlobContainerSasPermissions.Delete, expiresOnMinutes: 30, cancellationToken: cancellationToken);
```

Comment:
I would set the expiresOnMinutes to 5 mins instead of 30, since it's a more "desctructive" action to delete files than to create them. Can you also rename it to expiresInMinutes? Probably a typo by me

Reply:
Oh yeah, forgot to change it